### PR TITLE
FUSETOOLS2-1209 - provide opt-in telemetry

### PR DESCRIPTION
**Pull request by apupier**
minimal telemetry to collect number of times the extension is activated
and see the OS shares to help driving priorities

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -67,11 +68,16 @@ export function activate(context: ExtensionContext) {
 	window.onDidChangeActiveTextEditor((editor) =>{
 		toggleItem(editor, item);
 	});
-});
+
+	});
```
Comment (apupier) [inline src/extension.ts:72]: it seems there is an issue with indentation previously. To not have modification on a lot of unrelated lines i cahnged only one which is mitigating the issue.

**Review by bfitzpat — APPROVED** [review]
